src/changelog/.2.x.x/.release-notes.adoc.ftl

```diff
@@ -20,14 +20,21 @@
 
 <#if release.date?has_content>Release date:: ${release.date}</#if>
 
-This releases contains ...
+This minor release contains bug fixes, behavioral improvements, and a fully-fledged support for the GraalVM native image generation.
 
+[#release-notes-2-25-0-graalvm]
 === GraalVM reachability metadata
 
-Log4j Core and all its extension modules have been enhanced with embedded
+Log4j Core and all its extension modules have been enriched with embedded
 https://www.graalvm.org/latest/reference-manual/native-image/metadata/[GraalVM reachability metadata].
-This will allow the generation of GraalVM native images out-of-the-box, without any additional step necessary.
+This allows the generation of GraalVM native images out-of-the-box without any additional steps.
+See our xref:graalvm.adoc[GraalVM guide] for details.
 
-See our xref:graalvm.adoc[GraalVM guide] for more details.
+[#release-notes-2-25-0-PL-ex]
+=== Exception handling in Pattern Layout
+
+Exception handling in Pattern Layout went through a major rewrite.
+This effectively helped with fixing some bugs by matching the feature parity of all exception converters.
+Additionally, rendered stack traces are ensured to be prefixed with a newline, which used to be a whitespace in earlier versions.
```

Comment:
We should mention that during the rewrite we removed the `ansi` option to the pattern converters, which required an additional external dependency on an outdated (probably unsupported?) JAnsi 1.x library.

We can also drop the `ansi` option to the `%m` converter and announce it together. 99% of your user base does not have JAnsi 1.x on their stack.

Reply:
Fixed in 28a8325413ae8fd923d3174a080271cfb92625ad.